atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -90,10 +101,14 @@ private CassandraKeyValueServices() {
      * @param schemaMutationTimeMillis      time to wait for nodes' schema versions to match.
      * @param client                        Cassandra client.
      * @param unsafeSchemaChangeDescription description of the schema change that was performed prior to this check.
+     * @param availabilityRequirement Number of Cassandra nodes that must be reachable for schema consensus.
```

Comment:
not actionable: so the row above would suggest that we may want to align the description vertically, though I don't think that's necessary.